pyseir/deployment/model_conversion_factors.py

```diff
@@ -0,0 +1,102 @@
+import us
+import structlog
+from datetime import timedelta
+
+from pyseir import load_data
+
+log = structlog.get_logger()
+
+UNITY_SCALING_FACTOR = 1
+
+
+def _get_model_to_dataset_conversion_factors_for_state(state, t0_simulation, fips):
+    """
+    Return scaling factors to convert model hospitalization and model icu numbers to match
+    the most current values provided in combined_datasets.
+
+    Parameters
+    ----------
+    state
+    t0_simulation
+    fips
+
+    Returns
+    -------
+    convert_model_to_observed_hospitalized
```

Comment:
I actually like the names you have down below, ` model_to_observed_hospitalized_ratio, model_to_observed_icu_ratio`